<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player_window_ui</class>
 <widget class="QMainWindow" name="Player_window_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="playback_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_pause_btn">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>play_pause_btn</zorder>
   <zorder>playback_slider</zorder>
   <zorder>playback_slider</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_fIle">
    <property name="title">
     <string>&amp;FIle</string>
    </property>
    <addaction name="file_open"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="help_about"/>
   </widget>
   <addaction name="menu_fIle"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="file_open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="help_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
